.github/workflows/main.yml

```diff
@@ -80,3 +86,49 @@ jobs:
           --url="$(cat Docs/obj/url.txt)"
           --token="$GH_CHECK_STATUS_TOKEN"
       if: github.event_name != 'pull_request'
+
+  bundle:
+    name: bundle
+    needs: [build]
+    runs-on: ubuntu-18.04
+    steps:
+    - run: sudo apt-get install nuget
+    - uses: actions/checkout@master
+      if: github.event_name != 'pull_request'
+    - uses: actions/checkout@master
+      if: github.event_name == 'pull_request'
+      with:
+        ref: ${{ github.pull_request.head.sha }}
+    - uses: actions/download-artifact@master
+      with:
+        name: dist-bin
+        path: /tmp/nupkg
+    - run: |
+        targets=(netcoreapp3.1 net472 net462 net461)
```

Comment:
Shouldn't we add netstandard2.0?

Reply:
It would be better if we can support `netstandard2.0`, but a console app seems unable to be `netstandard2.0`…